mkdocs.yml

```diff
@@ -63,7 +63,6 @@ pages:
           - 'Multitenancy': 'cygnus-ngsi/installation_and_administration_guide/multitenancy.md'
           - 'Reliability': 'cygnus-ngsi/installation_and_administration_guide/reliability.md'
           - 'Performance tips': 'cygnus-ngsi/installation_and_administration_guide/performance_tips.md'
-          - 'NGSIv2 support': 'cygnus-ngsi/installation_and_administration_guide/ngsiv2_support.md'
```

Comment:
This document have sense when NGSIv2 notifications wasn't supported. However, now that it is, it can be removed.

NTC (informative)